runtime/doc/options.txt

```diff
@@ -52,7 +52,7 @@ achieve special effects.  These options come in three forms:
 			  'lines'
 			Warning: This may have a lot of side effects.
 
-						*:set-args* *E487* *E521*
+					*:set-args* *E487* *E521* *E5383*
```

Comment:
E5383 doesn't seems to be currently reachable because the type is already checked by `tv_to_optval()`, and it definitely cannot be thrown by `:set` because `:set` doesn't even call `set_option_value()`. Maybe it's better to remove this tag.

Reply:
Wait, it can be reached through `nvim_set_option_value` though, right?

Reply:
Actually API calls already prepend an E5555 error number, so maybe just remove this error number from the message?

Reply:
Actually many other API type mismatches don't have an error number either. So maybe just remove this error number from the message?

Reply:
Created a new PR #23942